api-datos/Dockerfile

```
@@ -0,0 +1,28 @@
+# Compila el servidor
+FROM golang:1.12.9 AS compila_go
+WORKDIR /workdir/cmd
+COPY ./cmd /workdir/cmd
+RUN go build -o ../bin/tablacruce ./tablacruce
```

No estoy seguro que acepte la versión larga:
```suggestion
RUN go build --output ../bin/tablacruce ./tablacruce
```

---

Parece que aquí no hay versión larga